src/main/java/uk/gov/companieshouse/mapper/CompanyOfficerMapper.java

```diff
@@ -0,0 +1,46 @@
+package uk.gov.companieshouse.mapper;
+
+import org.springframework.stereotype.Service;
+import uk.gov.companieshouse.api.model.officers.CompanyOfficerApi;
+import uk.gov.companieshouse.model.db.dissolution.DissolutionDirector;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Service
+public class CompanyOfficerMapper {
+    public boolean mapCompanyOfficersToIsOverHalfSelected(
```

Comment:
Boolean functions should start with `has` or `is` or `are` i.e. `areMajorityOfCompanyOfficersSelected`